controller/src/main/java/io/pravega/controller/server/SegmentHelper.java

```diff
@@ -107,6 +107,7 @@
             .put(WireCommands.ReadTable.class, ImmutableSet.of(WireCommands.TableRead.class))
             .put(WireCommands.ReadTableKeys.class, ImmutableSet.of(WireCommands.TableKeysRead.class))
             .put(WireCommands.ReadTableEntries.class, ImmutableSet.of(WireCommands.TableEntriesRead.class))
+            .put(WireCommands.FlushToStorage.class, ImmutableSet.of(WireCommands.FlushedStorage.class))
```

Comment:
I don't think this is correct. For the existing commands, we used `SegmentHelper` to send commands to the Segment Store because they belong to the regular protocol. But here you are mixing Admin-specific stuff to a class used by the controller. One option would be to create a new class in the admin cli package extending `SegmentHelper` where we can put all the necessary admin-specific commands, while keeping the already written functionality in `SegmentHelper`. Another option may be to put `SegmentHelper` in the `shared` package, as it is implementing a functionality that can be used by multiple modules in Pravega.

Reply:
Created a separate class `CommandSender` in admin-cli package and using it.